selfdrive/boardd/boardd.cc

```diff
@@ -18,6 +18,9 @@
 #include <cstdlib>
 #include <future>
 #include <thread>
+#ifdef __APPLE__
+#include <array>
```

Comment:
Should be no problem with always including this, right?

Reply:
yes, removed the #ifdef

Reply:
I think these should be alphabetical now :)